README.md

```diff
@@ -1,21 +1,70 @@
 # [Sendbird](https://sendbird.com) SyncManager for iOS
 
-[Sendbird](https://sendbird.com) SyncManager is a framework that caches and manages channels and messages of [Sendbird SDK](https://github.com/sendbird/sendbird-ios-framework). The SyncManager offers an event-based data management so that each view would see a single method by subscribing data event. And it stores the data into database(sqlite) which implements local caching for faster loading.
-
 [![Platform](https://img.shields.io/badge/platform-iOS-orange.svg)](https://cocoapods.org/pods/SendBirdSyncManager)
 [![Languages](https://img.shields.io/badge/language-Objective--C%20%7C%20Swift-orange.svg)](https://github.com/sendbird/sendbird-syncmanager-ios)
 [![CocoaPods](https://img.shields.io/badge/CocoaPods-compatible-green.svg)](https://cocoapods.org/pods/SendBirdSyncManager)
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![Commercial License](https://img.shields.io/badge/license-Commercial-brightgreen.svg)](https://github.com/sendbird/sendbird-syncmanager-ios/blob/master/LICENSE.md)
 
-## Documentation
-[SyncManager for iOS](https://docs.sendbird.com/ios/sync_manager_getting_started)
+## Table of contents
+
+  1. [Introduction](#introduction)
+  1. [Before getting started](#before-getting-started)
+  1. [Getting started](#getting-started)
+  1. [Implementation guide](#implementation-guide)
```

Comment:
all # 1?

Reply:
yep. i don't know why it works, but it works!

Reply:
![Screen Shot 2020-11-24 at 6 41 33 PM](https://user-images.githubusercontent.com/32758512/100076627-be684f80-2e84-11eb-85b2-dca2eabae167.png)